announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/tool/AnnouncementAction.java

```diff
@@ -2863,8 +2864,30 @@ else if (announceTo != null && announceTo.equals("groups"))
 					{
 						// availablity changed
 						eventTrackingService.post(eventTrackingService.newEvent(AnnouncementService.EVENT_ANNC_UPDATE_AVAILABILITY, msg.getReference(), true));
+
+						//SAK-44622:
```

Comment:
I wouldn't bother with the SAK number, to be honest. We can backtrack using git blame anyway, if we need to look up which JIRA introduced a change.

Reply:
done